views/apartments/show.erb

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title><% @apartment.title %> </title>
+</head>
+<body>
+  <p>
+
+    <a href class="song.">
```

Comment:
this `a` tag doesn't actually link to something. In order for it to link we need to have a value for it. `<a href="some_path" class="song">Link!</a>`